build/scripts/meta.yaml.schema

```diff
@@ -118,7 +118,19 @@
               },
               "memoryLimit": {
                 "type": "string",
-                "pattern": "[1-9][0-9]*([Gg]i|(Mm)i|(Kk)i|b)?"
+                "pattern": "^[1-9][0-9]*([Gg]i?|[Mm]i?|[Kk]i?|b)?$"
```

Comment:
Why don't you believe that some plugin may need 1Ti of memory :smile:

Reply:
Good catch with `^` and `$`. It makes sense to review all fields where we use regexps, like `image`.